### PR TITLE
Fix setup.py linux build

### PR DESCRIPTION
**Pull request by BrentHoltsclaw**

### CODE REVIEW

**Comment by hughsie** [comment]
Is building as non-root meant to be allowed? 

    ./setup.py build
    running build
    running build_py
    running build_ext
    building the linux driver
    creating /home/hughsie/Code/chipsec/build/lib.linux-x86_64-2.7/drivers
    error: could not create '/home/hughsie/Code/chipsec/build/lib.linux-x86_64-2.7/drivers': Permission denied

And as root, I get:

    building compression executables
    removing '/home/hughsie/Code/chipsec/build/lib.linux-x86_64-2.7/chipsec_tools/compression/bin' (and everything under it)
    error: can't copy '/home/hughsie/Code/chipsec/build/lib.linux-x86_64-2.7/chipsec_tools/compression/bin/Brotli': doesn't exist or not a regular file

**Comment by BrentHoltsclaw** [comment]
I am currently using setup.py build_ext -i opposed to build.  Is this acceptable?

**Comment by BrentHoltsclaw** [comment]
Build appears to be working now too

File: setup.py [hunk 1]
```diff
@@ -96,14 +97,13 @@ def _build_linux_compression(self):
...
-            dir_util.remove_tree(dst)
+            os.mkdir(dst)
         except:
             pass
-        os.mkdir(dst)
         for elf in elfs:
             self.copy_file(os.path.join(build_elf,"bin",elf),dst)
         # Clean up the build directory
-        dir_util.remove_tree(os.path.join(self.real_build_lib,"chipsec_tools"))
+        #dir_util.remove_tree(os.path.join(self.real_build_lib,"chipsec_tools"))
```
Comment (ErikBjorge) [inline setup.py]: Remove dead code

File: setup.py [hunk 2]
```diff
@@ -140,14 +140,13 @@ def _build_darwin_compression(self):
...
-            dir_util.remove_tree(dst)
+            os.mkdir(dst)
         except:
             pass
-        os.mkdir(dst)
         for exe in exes:
             self.copy_file(os.path.join(build_exe,"bin",exe),dst)
         # Clean up the build directory
-        dir_util.remove_tree(os.path.join(self.real_build_lib,"chipsec_tools"))
+        #dir_util.remove_tree(os.path.join(self.real_build_lib,"chipsec_tools"))
```
Comment (ErikBjorge) [inline setup.py]: More dead code to remove

**Comment by frinzell** [comment]
Just the copyright year.

**Comment by hughsie** [comment]
Works for me as non-root now, thanks!

File: setup.py (hunk 1 above)
Comment (BrentHoltsclaw) [reply setup.py]: done

File: setup.py (hunk 2 above)
Comment (BrentHoltsclaw) [reply setup.py]: done

**Pull request merged**